src/plugins/tabs/_screen-sm-max.scss

```diff
@@ -50,13 +59,19 @@
 .wb-tabs {
 	@extend %tabs-screen-sm-max-details; /* Only for backwards compatibility. Should be removed in v4.1. */
 
-	border-radius: $tablist-screen-sm-details-border-radius;
-	border-top-width: $tablist-screen-sm-details-border-width;
-  border-top-style: $tablist-screen-sm-details-border-style;
-  border-top-color: $tablist-screen-sm-details-border-color;
+	border: {
+		radius: $tablist-screen-sm-details-border-radius;
+		top: {
+			width: $tablist-screen-sm-details-border-width;
+			style: $tablist-screen-sm-details-border-style;
+			color: $tablist-screen-sm-details-border-color;
+		}
```

Comment:
Need to optimize this so that it's not top or bottom otherwise it loose it's flexibility